_documentation/02-designing-printer-drivers.md

```diff
@@ -14,7 +14,7 @@ __Table of Contents__
 
 <h2 id="introduction"> Introduction </h2>
 
-A Driver is an Interface between the user and the Operating System. Printing in Linux has move towards [Driverless Printing](https://openprinting.github.io/driverless/), i.e. we no longer need to download a driver for a printer. However, there are some problems with the current framework. For example, some printers(especially the old ones) that cannot handle ipp requests are devoid of driverless printing capability. Printer Application helps to address these issues. Kindly refer <a href="../01-printer-application/">Printer Applications - A new way to print in Linux</a> to learn more about Printer Application, it's working and benefits. 
+A driver is a code or data specific to a certain model or group of hardware devices, needed to make the hardware work with the hardware-model-independent independent code of the operating system. Printing in Linux has move towards [Driverless Printing](https://openprinting.github.io/driverless/), which means there is no need for any hardware-model-specific code or data. However, there are some problems with the current framework. For example, some printers(especially the old ones) that cannot handle IPP requests are devoid of driverless printing capability. Printer Applications help to address these issues. Kindly refer <a href="../01-printer-application/">Printer Applications - A new way to print in Linux</a> to learn more about Printer Applications, its working and benefits.
```

Comment:
With the "hardware-model-independent independent code" I actually did a mistake. There has to be only a single "independent". It must be simply "hardware-model-independent code". And there is also another mistake I discovered now. You wrote "Printing in Linux has move towards" but it must read "Printing in Linux has move**d** towards". Can you correct these two things.